ThingStub.cs

```diff
@@ -22,5 +24,13 @@ public int Zero()
         {
             return 0;
         }
+
+        public void TwoParameters(string a, Guid b)
+        {
+        }
+
+        public void FourParameters(string a, Guid b, string c, Guid d)
+        {
+        }
```

Comment:
Would the test be more valuable if we did something with the parameter? Could/should that be a second benchmark? I'm still waffling on the more vs less question...

Reply:
We could....but I think we want to test what the mock frameworks do without having callback or whatnot affect the tests themselves.